### PR TITLE
light styling fix and use of gray300 in address datatype

### LINKED ISSUE #5607: release blocking issue with styling
Looks like the styling changes have caused systemic changes to the light theme.

For example, the title labels in the tabs not selected are lighter now:
image

And it is really obvious in the About menu where the buttons/links all have essentially gone away:
image

Here is what happens if hover over the user manual button:
image

Regarding colors, may make sense to completely back out of those related changes in the styling and then just hard code the color where it was specifically needed in the address form stuff. Or else risk putting out a release with systemic color/contrast issues and seems quickest way to fix it.

### PR DESCRIPTION
**Pull request by bradymiller**
fixes #5607

### CODE REVIEW

File: interface/themes/colors/utilities/default_variables.scss [hunk 1]
```diff
@@ -52,16 +52,7 @@ $theme-colors: (
...
-  oe-midnightblue: $midnightblue,
-  gray100: $gray-100,
-  gray200: $gray-200,
-  gray300: $gray-300,
-  gray400: $gray-400,
-  gray500: $gray-500,
-  gray600: $gray-600,
-  gray700: $gray-700,
-  gray800: $gray-800,
-  gray900: $gray-900
```
Comment (sjpadgett) [inline interface/themes/colors/utilities/default_variables.scss:64]: Should be included and were there prior to new changes. These grays get used in some themes.

File: interface/themes/default-variables.scss [hunk 2]
```diff
@@ -50,16 +50,7 @@ $theme-colors: (
...
-  body-color: $body-color,
-  gray100: $gray-100,
-  gray200: $gray-200,
-  gray300: $gray-300,
-  gray400: $gray-400,
-  gray500: $gray-500,
-  gray600: $gray-600,
-  gray700: $gray-700,
-  gray800: $gray-800,
-  gray900: $gray-900
```
Comment (sjpadgett) [inline interface/themes/default-variables.scss:62]: yep should go.

File: interface/themes/oe-styles/style_light.scss [hunk 3]
```diff
@@ -5,8 +5,6 @@ $close-color: $black;
 $close-text-shadow: 0 1 0 $black;
 $gray: #cdcdcd;
 
-@import "../default-variables";
```
Comment (sjpadgett) [inline interface/themes/oe-styles/style_light.scss:8]: This is what broke light theme. I knew there was a reason this was patched out in past. Now we know why.

File: library/templates/address_list_display.php [hunk 4]
```diff
@@ -26,12 +26,12 @@
 
 <div id="<?php echo attr($table_id); ?>" class="row mt-3">
     <div class="table_text_addresses col-12">
-        <div class ="text_addresses_header label_custom pl-1 bg-gray300" style="text-align: left !important; padding-top: 0.1rem;">
+        <div class ="text_addresses_header label_custom pl-1" style="text-align: left !important; padding-top: 0.1rem;; background-color: var(--gray300)">
```
Comment (sjpadgett) [inline library/templates/address_list_display.php:29]: This works for me I guess. Still would have preferred setting a class in navmenu.scss(or core theme) as this is a useful shade.

**Review by sjpadgett — APPROVED** [review]
Thanks for testing and catching this ole eagle eye!

File: interface/themes/colors/utilities/default_variables.scss (hunk 1 above)
Comment (sjpadgett) [reply interface/themes/colors/utilities/default_variables.scss:64]: Actually just below in theme color array:
```
$theme-colors: (
  primary: $blue,
  secondary: $dark,
  success: $green,
  info: $cyan,
  warning: $yellow,
  danger: $red,
  light: $gray-100,
  dark: $gray-800,
  oe-darkest: $darkest,
  oe-darker: $darker,
  oe-dark: $dark,
  oe-mid: $mid,
  oe-midpale: $midpale,
  oe-pale: $pale,
  oe-midpalepale: $midpalepale,
  oe-palepale: $palepale,
  oe-palepurple: $palepurple,
  oe-paler: $paler,
  oe-midnightblue: $midnightblue,
  gray100: $gray-100,
  gray200: $gray-200,
  gray300: $gray-300,
  gray400: $gray-400,
  gray500: $gray-500,
  gray600: $gray-600,
  gray700: $gray-700,
  gray800: $gray-800,
  gray900: $gray-900
);
```

**Comment by bradymiller** [comment]
@sjpadgett , thanks for review. testing well and bringing in.

**Pull request merged**